nexus/api.go

```diff
@@ -145,6 +146,25 @@ func addCallbackHeaderToHTTPHeader(nexusHeader Header, httpHeader http.Header) h
 	return httpHeader
 }
 
+func addLinksToHTTPHeader(links []Link, httpHeader http.Header) error {
+	for _, link := range links {
+		httpHeader.Add(headerLink, encodeLink(link))
+	}
+	return nil
+}
+
+func getLinksFromHeader(httpHeader http.Header) ([]Link, error) {
+	var links []Link
+	for _, encodedLink := range strings.Split(strings.Join(httpHeader.Values(headerLink), ","), ",") {
```

Comment:
```suggestion
	for _, encodedLink := range httpHeader.Values(headerLink) {
```

I would expect Go is already doing the comma-delimited separation for you. Can you confirm?

Reply:
https://github.com/nexus-rpc/sdk-go/pull/17/files#r1728048912

Reply:
I verified using `curl ... -H "foo: a, b"` and this line of code in an http handler: `fmt.Println(r.Header.Values("foo")[0])`.

I got `a, b`

Reply:
:+1:, can ignore me here.

Reply:
Yes, I did the same, and Go does not auto-split the values.